parsers/src/test/scala/uk/gov/ons/addressIndex/crfscala/CrfFeatureTest.scala

```diff
@@ -0,0 +1,120 @@
+package uk.gov.ons.addressIndex.crfscala
+
+import org.scalatest.{FlatSpec, Matchers}
+import uk.gov.ons.addressIndex.crfscala.CrfFeatureAnalyser.CrfFeatureAnalyser
+import scala.util.Try
+
+class CrfFeatureTest extends FlatSpec with Matchers  {
+
+  object TestInstanceBoolean {
+    type tType = Boolean
```

Comment:
This is a little helper for the test; the idea is that i could C+P the block for the next type test and just change the type and value